src/menu/message.js

```diff
@@ -129,8 +129,9 @@ export function buildMessageEditorMenu(schema) {
   r.inlineMenu = [
     cut([r.toggleStrong, r.toggleEm, r.toggleCode, r.toggleLink]),
   ];
-  r.blockMenu = [cut([r.wrapBulletList, r.wrapOrderedList])];
-  r.fullMenu = r.inlineMenu.concat([[r.undoItem, r.redoItem]], r.blockMenu);
+  r.blockMenu = [cut([r.toggleBulletList, r.toggleOrderedList])];
+  r.fullMenu = [...r.inlineMenu, [r.undoItem, r.redoItem], ...r.blockMenu];
+  r.customMenu = [customMenuList.map(item => r[item])];
```

Comment:
The approach is correct here. However, the implementation needs work. Let us pair on this tomorrow @iamsivin